core/src/main/java/nl/nn/adapterframework/configuration/digester/ValidateAttributeRule.java

```diff
@@ -66,6 +67,8 @@ protected void handleAttribute(String name, String value, Map<String, String> at
 		}
 		if (m==null) { //validate if the attribute exists
 			addLocalWarning("does not have an attribute ["+name+"] to set to value ["+value+"]");
+		} else if(AnnotationUtils.findAnnotation(m, SuppressAttribute.class) != null) {
+			addLocalWarning("is unable to set attribute ["+name+"] to value ["+value+"]");
```

Comment:
'unable' is hier niet het goede woord. Misschien: "attribute [] is protected, cannot be set from configuration".
Dat is dan meteen een alternatieve naam voor @SuppressAttribute: @ProtectedAttribute. Dat sluit aan bij de programmeer-betekenis van 'Protected'